tools/eslint/package.json

```diff
@@ -14,6 +14,7 @@
     "eslint-plugin-jest": "25.3.4",
     "eslint-plugin-jest-playwright": "0.2.1",
     "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-node": "11.1.0",
```

Comment:
Where do we use this?

Reply:
In PR #2853, we added a number of eslint overrides, and they addressed all the eslint errors we were getting for the Docusaurus app. Hence, the overrides in this PR are identical to what we had previously (just located in `src/docs` as opposed to `/`). 

For some reason, I am no longer getting any eslint errors for the Docusaurus app on master right now. I don't know if this PR is even needed...

Reply:
What if you introduce an eslint error, does it catch that?

Reply:
@menghif we use `eslint-plugin-node` here

https://github.com/Seneca-CDOT/telescope/blob/36f2307c949a5f943037cfcd979680430f1908f1/src/docs/.eslintrc.js#L25-L26

or else we get errors like these
![2022-02-22 13_24_38-Window](https://user-images.githubusercontent.com/36822198/155195417-b8e5df61-f5e4-4690-842d-fe5bb2f38d39.png)